src/features/bookings/components/BookingDetailsCancelButton.native.test.tsx

```diff
@@ -102,6 +103,29 @@ describe('<BookingDetailsCancelButton />', () => {
       )
     ).toBeTruthy()
   })
+
+  it('should display button if confirmationDate is null and an expiration date message if isBookingIsDigital', () => {
+    const booking = { ...bookingsSnap.ongoing_bookings[0] }
+    booking.confirmationDate = null
+    booking.stock.offer.isDigital = true
+
+    const { queryByTestId, queryByText } = renderBookingDetailsCancelButton(booking)
+    const expirationDateMessage = 'Ta réservation expirera le 17/03/2021'
+
+    expect(queryByTestId('Annuler ma réservation')).toBeTruthy()
+    expect(queryByText(expirationDateMessage)).toBeTruthy()
+  })
+
+  it('should display only an expiration date message if is not still cancellable and if the booking is digital', () => {
```

Comment:
should display only an expiration date message when the booking is digital and is not still cancellable